data/migrations/versions/5b8dc452f5c3_add_tag_immutability_column_and_logentry_kinds.py

```diff
@@ -0,0 +1,74 @@
+"""add tag immutability column
+
+Revision ID: 5b8dc452f5c3
+Revises: a32e17bfad20
+Create Date: 2023-05-21 13:13:10.565161
```

Comment:
Does the date need to be updated?

Reply:
No. That's just the date the file was generated